FullBackgroundAudio/app/src/main/java/com/jwplayer/demo/fullbackgroundaudio/App.java

```diff
@@ -0,0 +1,34 @@
+package com.jwplayer.demo.fullbackgroundaudio;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+public class App extends Application {
+    //Notification related properties
+    public static final String CHANNEL_ID = "JW";
+    private static final String CHANNEL_NAME = "JW Notifications";
+    private static final String CHANNEL_DESCRIPTION = "This is the default notification channel for the JW Player";
+    private static final int CHANNEL_IMPORTANCE = NotificationManager.IMPORTANCE_LOW;
+    public static final int NOTIFICATION_ID = 1;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        createNotificationChannel();
+    }
+
+    private void createNotificationChannel(){
```

Comment:
Can we add more comments here on why we are creating a notification channel, i.e., to support playback control for the background app

Reply:
Comments added